os_info/src/freebsd/mod.rs

```diff
@@ -30,7 +30,16 @@ fn get_os() -> Type {
         .expect("Failed to get OS");
 
     match str::from_utf8(&os.stdout).unwrap() {
-        "FreeBSD\n" => Type::FreeBSD,
+        "FreeBSD\n" => {
+            let check_hardening = Command::new("sysctl")
+                .arg("hardening.version")
+                .output()
+                .expect("Failed to check if is hardened");
+            match str::from_utf8(&check_hardening.stderr).unwrap() {
```

Comment:
I think it would be better to return `Type::FreeBSD` here instead of panic.

Reply:
Would the solution here to be to drop the expected statement? From the match, it should default to returning FreeBSD.

Reply:
![Screenshot 2022-03-10 214330](https://user-images.githubusercontent.com/33560895/157809433-23037ec6-63cc-4d37-9dc2-2dbd4829d2fb.png)

Built in release and double checked. Nothing indicates a panic on output and returns FreeBSD on my FreeBSD VPS.

Reply:
Technically `str::from_utf8(...).unwrap()` can panic, but nevermind: there is one more `from_utf8` call near, so I'm going to fix both before the next release.

Reply:
Oh gotcha. I was under the assumption that if a failure happened, the `_ =>` case would be hit.